src/ui.message.js

```diff
@@ -74,6 +82,10 @@ var _ = Mavo.UI.Message = $.Class({
 			$.remove(this.element);
 			this.closed.resolve(resolve);
 		});
+	},
+
+	nonAnimatedClose: function() {
+		$.remove(this.element);
```

Comment:
We don't want a second function just to prevent animation!
We already have the option stored, we can check it. Or we can just pass a duration of `0` to `$.transition()`. Not to mention that this function doesn't even resolve `this.closed`.